recipes/cfn-flip/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "cfn-flip" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/awslabs/aws-cfn-template-flip/archive/{{ version }}.tar.gz
+  sha256: 2ea2d115a69416acd577c2d97c7731d1a8fc313544831106e756ef22995c73a1
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - cfn-flip=cfn_flip.main:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - click
+    - pip
+    - python
+    - pyyaml >=4.1
```

Comment:
Are click and pyyaml really needed to install the package?

Reply:
It seems like they are only run deps

Reply:
thank you. changes have been made to have click and pyyaml only as run deps.